<?xml version="1.0" encoding="UTF-8"?>
<!--
 * @license
 * Copyright 2018 AI Lab - Telkom University. All Rights Reserved.
 * Licensed under the Apache License, Version 2.0 (the "License")
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * =============================================================================
-->
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>639</width>
				<height>565</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<property name="styleSheet">
			<string notr="true"/>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QGroupBox" name="groupBox">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>80</y>
						<width>621</width>
						<height>381</height>
					</rect>
				</property>
				<property name="sizePolicy">
					<sizepolicy hsizetype="Preferred" vsizetype="Preferred">
						<horstretch>0</horstretch>
						<verstretch>0</verstretch>
					</sizepolicy>
				</property>
				<property name="title">
					<string>Video</string>
				</property>
				<widget class="QWidget" name="ImgWidget" native="true">
					<property name="geometry">
						<rect>
							<x>10</x>
							<y>20</y>
							<width>601</width>
							<height>351</height>
						</rect>
					</property>
				</widget>
			</widget>
			<widget class="QPushButton" name="startButton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>10</y>
						<width>300</width>
						<height>61</height>
					</rect>
				</property>
				<property name="text">
					<string>Start video</string>
				</property>
			</widget>
			<widget class="QPushButton" name="stopButton">
				<property name="geometry">
					<rect>
						<x>320</x>
						<y>10</y>
						<width>300</width>
						<height>61</height>
					</rect>
				</property>
				<property name="text">
					<string>Stop video</string>
				</property>
			</widget>
			<widget class="QPushButton" name="registerButton">
				<property name="geometry">
					<rect>
						<x>220</x>
						<y>470</y>
						<width>191</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Register</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="newLabel">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>470</y>
						<width>181</width>
						<height>41</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>12</pointsize>
					</font>
				</property>
			</widget>
			<widget class="QCheckBox" name="useSpoof">
				<property name="geometry">
					<rect>
						<x>430</x>
						<y>480</y>
						<width>121</width>
						<height>21</height>
					</rect>
				</property>
				<property name="text">
					<string>Use Spoof Detection</string>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>639</width>
					<height>21</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>


<!-- @author ANDITYA ARIFIANTO, AI LAB - 2018 -->